Makefile

```diff
@@ -435,6 +435,7 @@ generate-files:
 	python3 scripts/generate_serialization.py
 	python3 scripts/generate_enum_util.py
 	python3 tools/pythonpkg/scripts/generate_connection_code.py
+	python3 tools/pythonpkg/scripts/generate_functions_code.py
```

Comment:
As you mentioned, this fails because it relies on the `duckdb` module being present.
Either we need to change the CI run that uses `make generate-files` to first build python duckdb, I think this makes sense if there is only one workflow that does this.

Reply:
Yeah agree, efficient if the build only happens in one place.

[Here](https://github.com/duckdb/duckdb/discussions/12134#discussioncomment-9556222) you mentioned, that the plan is to move `pythonpkg` into a separate repo which would probably help for this. In the meantime, a pragmatic solution could be that I:
* deactivate the execution of `generate_functions_code.py` in the `CodeQuality` workflow
* add it, incl. the `git diff` check, in the `Python 3.9 Linux` workflow after the package is built

This way, we'd at least still have something that checks that `func.py` is kept in sync. Thoughts?